From phil@ipom.com Mon Dec 26 23:09:34 2005
Message-ID: <43B0E778.6000508@ipom.com>
Date: Mon, 26 Dec 2005 23:04:24 -0800
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>, Alexandre Duret-Lutz <adl@gnu.org>
Subject: USB: usb-storage support for SONY DSC-T5 still camera


From: Alexandre Duret-Lutz <adl@gnu.org>

I've been offered a nice Sony DSC-T5 digital camera, with a USB connection.
Unfortunately it is not recognized by Linux 2.6.14.4's usb-storage.

With the following change I'm able to mount and read my pictures:

Signed-off-by: Phil Dibowitz <phil@ipom.com>

---
 drivers/usb/storage/unusual_devs.h |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -429,11 +429,11 @@ UNUSUAL_DEV(  0x054c, 0x0010, 0x0106, 0x
 		US_FL_SINGLE_LUN | US_FL_NOT_LOCKABLE | US_FL_NO_WP_DETECT ),
 
 /* This entry is needed because the device reports Sub=ff */
-UNUSUAL_DEV(  0x054c, 0x0010, 0x0500, 0x0500, 
-               "Sony",
-               "DSC-T1", 
-               US_SC_8070, US_PR_DEVICE, NULL,
-               US_FL_SINGLE_LUN ),
+UNUSUAL_DEV(  0x054c, 0x0010, 0x0500, 0x0600,
+		"Sony",
+		"DSC-T1/T5",
+		US_SC_8070, US_PR_DEVICE, NULL,
+		US_FL_SINGLE_LUN ),
 
 
 /* Reported by wim@geeks.nl */
